src/components/time/ReadableTime.js

```diff
@@ -51,20 +58,21 @@ const ReadableTime = ({
     } else if (showWeekday) {
         // e.g. Monday, Oct 5, 2018
         dateMessage = messages.eventTimeWeekdayLong;
-        weekday = <FormattedDate value={timestamp} weekday="long" />;
+        weekday = intl.formatDate(timestamp, { weekday: 'long' });
```

Comment:
using intl formatting here in order to uppercase string - not able to achieve this using the `Formatted` components itself

Reply:
👍